middleware/http/wasm/example/go.mod

```diff
@@ -2,6 +2,6 @@ module github.com/dapr/components-contrib/middleware/wasm/example
 
 go 1.19
 
-replace github.com/dapr/dapr => github.com/dapr/dapr v1.9.4-0.20221213222644-64d468a693f1
+replace github.com/dapr/dapr => github.com/mukundansundar/dapr v0.7.1-0.20221215171955-932ed4d7c0c8
```

Comment:
There are lots of people who are constantly adding replaces to add their own forks... We probably need to find a better way to manage this. CC: @berndverst

Reply:
For example: 
`middleware/wasm` does not depend on `dapr/dapr`, but also introduces. How do we use the `go` command to remove invalid replace.

![image](https://user-images.githubusercontent.com/20457624/208585521-217077af-b672-4ad0-9fd9-c525fccfb814.png)